lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -641,6 +650,18 @@
     comparisons:
       '*': ignore  # by default, ignore *all* options (including image)
       env: strict   # except for environment variables; there, we want to be strict
+
+- name: Start container with healthstatus
+  docker_container:
+    name: my_container
+    image: image_name
+    state: started
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost"]
```

Comment:
Maybe you should add a short comment explaining what this test achieves. I.e. something like `Tries to make a HTTP GET to localhost. If this fails or timeouts, the healthcheck fails.` (if that's correct).